lib/rules/rule-nested-empty-line-before/README.md

```diff
@@ -152,6 +152,35 @@ The following patterns are *not* considered warnings:
 }
 ```
 
+### `except: ["after-rule"]`
+
+Reverse the primary option if the rule is the first after a property.
+
+For example, with `"never"`:
```

Comment:
"For example, with `"always"`:"

Once you switch to `opts.rule.prev().type === "rule"` you'll hopefully see why this should be `"always"` and not `"never"`